detectree2/models/predict.py

```diff
@@ -1,64 +1,46 @@
+"""Generate predictions."""
```

Comment:
the only change in this one is to import defaultpredictor1 from custom_nms which is a defaultpredictor + proprocessing using mask-nms. The other conflicts are from that my version is not updated and should not be changed